src/components/app/details/cicdHistory/Sidebar.tsx

```diff
@@ -47,6 +48,9 @@ const Sidebar = React.memo(({ type, filterOptions, triggerHistory, hasMore, setP
     const selectedFilter = filterOptions?.find(
         (filterOption) => filterOption.value === (type === HistoryComponentType.CI ? pipelineId : envId),
     )
+
+    const _filterOptions = filterOptions?.filter((filterOption) => !filterOption.deploymentAppDeleteRequest )
```

Comment:
This is required in case of Deployment history only so there should be a check for that otherwise it will impact the build history as well

Reply:
It should not impact Build an deploy

Reply:
Done